### PR TITLE
Proposed update to issue template

### PR DESCRIPTION
**Pull request by karthik**
Based on issue at https://github.com/ropensci/onboarding-meta/issues/68

### CODE REVIEW

File: issue_template.md [hunk 1]
```diff
@@ -33,32 +45,36 @@ Confirm each of the following by checking the box.  This package:
...
 - [ ] has a test suite.
-- [ ] has continuous integration, including reporting of test coverage, using services such as Travis CI, Coveralls and/or CodeCov.
-- [ ] I agree to abide by [ROpenSci's Code of Conduct](https://github.com/ropensci/onboarding/blob/master/policies.md#code-of-conduct) during the review process and in maintaining my package should it be accepted.
+- [ ] has continuous integration, including reporting of test coverage using services such as Travis CI, Coveralls and/or CodeCov.
 
-#### Publication options
+## Publication options
 
 - [ ] Do you intend for this package to go on CRAN?  
 - [ ] Do you wish to automatically submit to the [Journal of Open Source Software](http://joss.theoj.org/)? If so:
```
Comment (sckott) [inline issue_template.md]: is it actuall automatic? i thought they had to submit manually to JOSS?

File: issue_template.md [hunk 2]
```diff
@@ -1,29 +1,41 @@
-### Summary
-
--   What does this package do? (explain in 50 words or less):
+Submitting Author: Name (@github_handle)  
+Repository:
```
Comment (sckott) [inline issue_template.md]: maybe specify url, `Repository URL: `

File: issue_template.md [hunk 3]
```diff
@@ -1,29 +1,41 @@
-### Summary
-
--   What does this package do? (explain in 50 words or less):
+Submitting Author: Name (@github_handle)  
+Repository:   
+Version submitted:
```
Comment (sckott) [inline issue_template.md]: depending on the decision on https://github.com/ropensci/onboarding-meta/issues/70 do we have submitters give the url to the git tagged version?

File: issue_template.md [hunk 4]
```diff
@@ -1,29 +1,41 @@
...
--   Are there other R packages that accomplish the same thing? If so, how does
-yours differ or meet [our criteria for best-in-category](https://github.com/ropensci/onboarding/blob/master/policies.md#overlap)?
+-   Are there other R packages that accomplish the same thing? If so, how does yours differ or meet [our criteria for best-in-category](https://github.com/ropensci/onboarding/blob/master/policies.md#overlap)?
 
--   If you made a pre-submission enquiry, please paste the link to the corresponding issue, forum post, or other discussion, or @tag the editor you contacted.
+-   If you made a pre-submission enquiry, please paste the link to the corresponding issue, forum post, or other discussion, or @tag the editor you contacted.
 
-### Requirements
 
 Confirm each of the following by checking the box.  This package:
```
Comment (sckott) [inline issue_template.md]: it looks like this section is under the __Scope__ heading

File: issue_template.md [hunk 5]
```diff
@@ -1,29 +1,41 @@
-### Summary
-
--   What does this package do? (explain in 50 words or less):
+Submitting Author: Name (@github_handle)  
+Repository:   
+Version submitted:   
+Editor: TBD  
+Reviewer 1: TBD  
+Reviewer 2: TBD  
+Archive: TBD
```
Comment (sckott) [inline issue_template.md]: will it be clear to submitters what __Archive__ means?

File: issue_template.md (hunk 1 above)
Comment (karthik) [reply issue_template.md]: I didn't change any of the text from our existing template. I just reformatted it. This is what our current template says. Example:

File: issue_template.md (hunk 2 above)
Comment (karthik) [reply issue_template.md]: Sure. But even if it were just repository, what else could it be?

File: issue_template.md (hunk 3 above)
Comment (karthik) [reply issue_template.md]: Sounds fine. For now, we still ask for a URL and this update just reformats where in the workflow we ask for it. But I agree that with the URL + Version, we can ensure that reviewers are not chasing a moving target.

File: issue_template.md (hunk 4 above)
Comment (karthik) [reply issue_template.md]: Good point. I meant to add technical checks or something like that. Do you have a suggestion for a sub-heading?

File: issue_template.md (hunk 5 above)
Comment (karthik) [reply issue_template.md]: Yes. and we can explain it in the docs. Zenodo archive makes it too prescriptive. Again in the docs we can explain in more details what it means to archive. Also in the initial template all of this is marked as TBD which means that no one will have to worry about filling this out.

File: issue_template.md (hunk 4 above)
Comment (sckott) [reply issue_template.md]: technical checks sounds good

File: issue_template.md [hunk 6]
```diff
@@ -1,29 +1,41 @@
...
 
-[e.g., "data extraction, because the package parses a scientific data file format"]
+	- [ ] data retrieval
+	- [ ] data extraction
+	- [ ] database access
+	- [ ] data munging
+	- [ ] data deposition
+	- [ ] reproducibility
+	- [ ] geospatial data
+	- [ ] text analysis
```
Comment (noamross) [inline issue_template.md]:
```suggestion
	- [ ] text analysis
	
	- Explain how the and why the package falls under these categories (briefly, 1-2 sentences):
```

File: issue_template.md [hunk 7]
```diff
@@ -1,29 +1,41 @@
...
 ```
 
 ```
 
--   URL for the package (the development repository, not a stylized html page):
 
-- Please indicate which category or categories from our [package fit policies](https://github.com/ropensci/onboarding/blob/master/policies.md#package-fit) this package falls under ***and why**(? (e.g., data retrieval, reproducibility. If you are unsure, we suggest you make a pre-submission inquiry.):
+## Scope 
+
+- Please indicate which category or categories from our [package fit policies](https://ropensci.github.io/dev_guide/policies.html#package-categories) this package falls under ***and why**(? (Please check an appropriate box below. If you are unsure, we suggest you make a pre-submission inquiry.):
```
Comment (noamross) [inline issue_template.md]:
```suggestion
- Please indicate which category or categories from our [package fit policies](https://ropensci.github.io/dev_guide/policies.html#package-categories) this package falls under: (Please check an appropriate box below. If you are unsure, we suggest you make a pre-submission inquiry.):
```

File: issue_template.md (hunk 6 above)
Comment (noamross) [reply issue_template.md]: Suggesting this change because its not clear where and how the author should explain scope.

**Comment by karthik** [comment]
Version updated with the comments above [is here](https://github.com/ropensci/onboarding/blob/d50403c822e67125a6afc554abccdada4898ceb6/issue_template.md). If I can get a LGTM from both of you, I'll merge into master. 🙏 .

**Comment by noamross** [comment]
LGTM

**Comment by sckott** [comment]
looks good

**Pull request merged**